DataFormats/GEMRecHit/interface/ME0Segment.h

```diff
@@ -72,7 +73,10 @@ class ME0Segment final : public RecSegment {
 
   float time() const    { return theTimeValue; }
   float timeErr() const { return theTimeUncrt; }
-    
+
+  float deltaPhi() const { return theDeltaPhi; }
+  static float computeDeltaPhi(const ME0Chamber * chamber, const LocalPoint& position, const LocalVector& direction );
```

Comment:
@ianna It would be used under: RecoMuon/GlobalMuonProducer/src/GlobalMuonProducer.h and the related classes when building RecoMuons. Technically it should also be used in ME0SegmentMatcher. 

I couldn't really think of a good place to put this function, and now that I think about it the ME0Segment isn't a very good place for it at all. I am open to any suggestion!

Thanks!